openedx/core/djangoapps/util/helpers.py

```diff
@@ -0,0 +1,48 @@
+"""
+Common helpers methods for django apps.
+"""
+
+import logging
+
+from django.core.exceptions import ImproperlyConfigured
+from provider.oauth2.models import AccessToken, Client
+from provider.utils import now
+
+
+log = logging.getLogger(__name__)
+
+
+def get_id_token(user, client_name):
+    """Generates a JWT ID-Token, using or creating user's OAuth access token.
+
+    Arguments:
+        user (User Object): User for which we need to get JWT ID-Token
+        client_name (unicode): Name of the OAuth2 Client
+
+    Returns:
+        String containing the signed JWT value or raise the exception
+        'ImproperlyConfigured'
+    """
+    # TODO: there's a circular import problem somewhere which is why we do the oidc import inside of this function.
```

Comment:
I don't think this is still a problem, I was waiting till I fix the `edxnotes` tests. I will update this.